src/redmine-net-api/Types/IssueRelation.cs

```diff
@@ -185,11 +189,15 @@ public override void ReadJson(JsonReader reader)
         IssueRelationType ReadIssueRelationType(JsonReader reader)
         {
             var enumValue = reader.ReadAsString();
-            if (short.TryParse(enumValue, out short enumId))
+            if (!enumValue.IsNullOrWhiteSpace())
             {
-                return (IssueRelationType)enumId;
+                if (short.TryParse(enumValue, out short enumId))
+                {
+                    return (IssueRelationType)enumId;
+                }
+                return (IssueRelationType)Enum.Parse(typeof(IssueRelationType), enumValue, true);
             }
-            return (IssueRelationType)Enum.Parse(typeof(IssueRelationType), enumValue, true);
+            return default;
```

Comment:
if we use default for enums then the default value of the enums should be None.

Reply:
Introducing a enumeration value of `None` would be misleading for users, which are creating relations. Maybe I can throw a exception instead, since the relation_type shouldn't be empty or null.

Reply:
I think is better with default state of enum None or Undefined or Unknown. Exceptions can be thrown on serialization.